interface/themes/style_manila.scss

```diff
@@ -9,7 +10,7 @@ body {
 	/* set the base font and size for all DOM children */
 	font-family: 'source_sans_pro','FontAwesome';
 	content: "\f0c4";
-	font-size: 12px;  /* set the base font size for all DOM children */
+	font-size: 16px;  /* set the base font size for all DOM children */
```

Comment:
Lets make sure this font size change does not break the Eye Form.

Reply:
Eye form looks fine

![2018-06-04 08_25_54-eye](https://user-images.githubusercontent.com/12654295/40947411-65ba4932-6817-11e8-948f-edf28d77584d.png)